packages/framework-integration-tests/integration/providers/local/setup.ts

```diff
@@ -1,14 +1,9 @@
 import { start } from '../../../src/deploy'
-import { ChildProcess, execSync } from 'child_process'
+import { ChildProcess } from 'child_process'
```

Comment:
I discovered that we have the package [`child-process-promise`](https://www.npmjs.com/package/child-process-promise) among our dependencies, which are wrappers around `child_process` functions without explicitly using promisify in our code.

Reply:
Not really sure what you mean here, do you want me to avoid using the `execSync` function?

Reply:
No, just mentioning that we have this other way to import `exec` that is "promisified" by default:

```typescript
import { exec } from 'child-process-promise'

...
await exec('ls -la') // You have the option to wait for the response using await
```